static/app/data/platformCategories.tsx

```diff
@@ -3,6 +3,15 @@
 
 import {t} from 'sentry/locale';
 
+export enum PlatformCategory {
+  frontend,
```

Comment:
enum values are typically denoted with all caps, i.e `FRONTEND`, `MOBILE`, we should change this here to be consistent with the rest of the codebase

Reply:
We should have a linting rule for this

Reply:
True, you should suggest it to the frontend channel! 😛